src/platform_impl/windows/monitor.rs

```diff
@@ -226,7 +226,7 @@ impl MonitorHandle {
         // EnumDisplaySettingsExW can return duplicate values (or some of the
         // fields are probably changing, but we aren't looking at those fields
         // anyway), so we're using a BTreeSet deduplicate
-        let mut modes = BTreeSet::<RootVideoModeHandle>::new();
+        let mut modes = Vec::new();
```

Comment:
I think we may still need to deduplicate here?

Reply:
Ah, good catch!

Reply:
Done, though I believe that we should actually not de-duplicate here but instead expose the underlying type with an extension specific trait; a matter for another time.